project-chip/connectedhomeip
# [Telink]: Add posibility to configure CHIP shell over Kconfig

#### Problem
- Absence of possibility to configure CHIP shell for Telink devices

#### Change overview
- Modify config/telink/chip-module/CMakeLists.txt with proper management of CONFIG_CHIP_LIB_SHELL
- Modify examples/lighting-app/telink/prj.conf with CONFIG_CHIP_LIB_SHELL=y

#### Testing
Manually. Steps:
1. Build and Flash lighting-app example
2. Connect to Matter device via uart 
3. Check if shell appeaser
4. Try several shell commands
